examples/BART/BART_introduction.ipynb

```diff
@@ -0,0 +1,557 @@
+{
```

Comment:
<p>You need to add a target at the top of the notebook too so it can be referenced from other notebooks or the versioned docs: https://docs.pymc.io/en/latest/contributing/jupyter_style.html#first-cell</p><p> </p><p>I think the notebook is clear and beginner friendly, and would change the category to beginner. It feels a bit strange to have introduction in the title but be intermediate. I ended up commenting only on formatting and one code issue.</p><p> </p><p>This is still in early discussion but we were considering with Martina to use 2 categories for notebooks, level and type one. The type one would be one of the 4 types defined in diataxis. This notebook imo fits quite well the explanation type: https://diataxis.fr/explanation/#the-language-of-explanation. Given that you are planning more notebooks on the topic, would that division and structure help? Do you think it would be useful for users?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/259/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Thanks for the feedback.

I am not sure about the criteria used for beginner, intermediate or advance categories. For example this notebook is and introduction to BART but assumed reader already knows Bayesian statistics and PyMC. So is beginners with respect to BART but non beginners with respect to Bayesian modeling.

Having more structure will give us a little bit more of work. Buy I think that will be beneficial for users, and even for us, that way it will be easier to spot missing documentations or areas than need more attention to get a balanced documentation. Also structure is very helpful for the autodidactic, that not only needs to read some specific documentation to solve a particular problem but want to use the documentation as part of a large learning plan.

Reply:
Not so sure either myself about those levels either, we should probably define them somewhere (cc @martinacantaro) and not worry about those too much until then.

If you find the diataxis structure useful, add `explanation` as second category (and then choose one category and label the other posts in the same manner). Several parts of the versioned docs are heavily inspired by diataxis so I think it will be good to try to follow it everywhere (to some extent)